unitTests/ui/ResponsiveComponents/ResponsiveDropdownHeaderTest.ts

```diff
@@ -27,6 +27,18 @@ export function ResponsiveDropdownHeaderTest() {
       expect(responsiveDropdownHeader.element.hasClass(expectedToBeRemovedClass)).toBe(false);
     });
 
+    it('adds hide css class when hide is called', () => {
```

Comment:
I think mentioning the class in the description is less vague. e.g. `when #hide is called, it adds coveo-hidden css class`

Similarly for the test on line 36, from the test it looks like there is no active class, but rather we remove the hidden class, so:
`when #show is called, it removes the coveo-hidden css class`